src/libraries/System.Net.Quic/src/System.Net.Quic.csproj

```diff
@@ -83,6 +121,14 @@
     <Reference Include="System.Threading.Channels" />
   </ItemGroup>
 
+  <ItemGroup>
+    <ProjectReference Include="$(LibrariesProjectRoot)System.Security.Cryptography.OpenSsl\src\System.Security.Cryptography.OpenSsl.csproj" />
```

Comment:
Do we still need this with all the added interop?

Reply:
Yes, we do, the interop uses public classes from this project. It doesn't build without it and the same it's referenced in S.N.Http.